optimize/util/optimize-commons/src/main/java/io/camunda/optimize/service/db/es/schema/TransportOptionsProvider.java

```diff
@@ -35,6 +37,18 @@ public RequestOptions getRequestOptions() {
     return optionsBuilder.build();
   }
 
+  public TransportOptions getTransportOptions() {
+    final DefaultTransportOptions.Builder optionsBuilder = new DefaultTransportOptions.Builder();
+    //    if (!customHeaderSuppliers.isEmpty()) {
+    //      customHeaderSuppliers.forEach(
+    //          headerFunction -> {
+    //            final CustomHeader customHeader = headerFunction.get();
+    //            optionsBuilder.addHeader(customHeader.getHeader(), customHeader.getValue());
+    //          });
+    //    }
```

Comment:
I think we can even delete this. There is no plan to bring the custom header plugin to C8 within the scope of Optimize

Reply:
fixed